compiler+runtime/test/jank/cpp/try/pass-catch-cpp-std-runtime-error.jank

```diff
@@ -0,0 +1,13 @@
+;; Test catching std::runtime_error
```

Comment:
This test is redundant with the `bad_cast`, `invalid_argument`, `logic_error`, and `out_of_range` tests. We don't need all five.